sashimi-webapp/src/database/sql-related/sqlCommands.js

```diff
@@ -0,0 +1,261 @@
+import StringManipulator from '../stringManipulation';
+
+import exceptions from '../exceptions';
+
+import constants from '../constants';
+
+import DateTime from '../generated-data/dateTime';
+
+const alasql = require('alasql');
```

Comment:
Hmm.. That's odd, I changed this line to use import and I was able to pass the database tests for this pull request. 

What error did you see on your side?